imports/ui/components/molecules/logo/LogoSettings.js

```diff
@@ -0,0 +1,73 @@
+import React, { Fragment, useState } from 'react'
+import styled from 'styled-components'
+
+// Atoms
+import { ColorSwatch } from '../../atoms/logo/ColorSwatch'
+import { ColorPicker } from '../../atoms/logo/ColorPicker'
+import { FontPicker } from '../../atoms/logo/FontPicker'
+import { SubText } from '../../atoms/text/SubText'
+
+// Styles
+const SettingsWrap = styled.div`
```

Comment:
In the existing project, we use the more general approach with _Row_ and _Column_ for this kind of layout logic.